app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controllers/topics")
+app.use((err, req, res, next) => {
+    console.error(err)
+    if (err.code === 'ECONNREFUSED') {
```

Comment:
we should not be getting this error so it should not need to be handled - this is probably to do with your `.env` files